src/components/PopupWithForm.js

```diff
@@ -30,22 +32,20 @@ class PopupWithForm extends Popup {
   }
 
   setEventListeners() {
-    super.addEventListeners();
+    super.setEventListeners();
     this._popupForm.addEventListener("submit", (event) => {
       event.preventDefault();
       this._handleFormSubmit(this._getInputValues());
-      this.close();
     });
   }
 
   open() {
     super.open();
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change it inside and only call it with `super`. The parent methods are available in any inheritor of the class.